### ISSUE TITLE
how to export a build for my coworkers to use?

### ISSUE THREAD

**Issue by LowellHuawei** [preesm/preesm#130]
Some of the problems our team has experienced with the released Preesm builds seem to have gone away in the developer branch version that Antoine helped me build.

How can I export my built Preesm to for my co-workers to use?

**Comment by antoine-morvan** [comment]
There is some doc here : https://github.com/preesm/preesm/blob/develop/releng/README.md

It is out dated, but the maven process to build the zip archives should work.

**Comment by LowellHuawei** [comment]
Thanks!

**Comment by LowellHuawei** [comment]
this seems to have worked.  Thanks for the assistance.

Perhaps the instructions should say how to find the actual sub-directory where the built executable environment can be found?  I used find . -name eclipse.exe after the build to find the x86_64 directory where the code was deposited.    Also, the build fails do to lack of the server to host the output on -- the sourceforge server.  This is not important to me, but it took me a while to realize that the failure didn't matter.  Perhaps a note about that in the build instructions would help.

But I am closing the issue since it all worked and I have been able to distribute my built executables.  Thanks again for the timely assistance!
